src/Raven.Client/Http/AggressiveCacheMode.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Raven.Client.Http
+{
+    public enum AggressiveCacheMode
+    {
+        Unknown = 0,
```

Comment:
Do we need `Unknown` ?

Better to switch to
- `Live`
- `Passive` maybe?

Reply:
We typically have _something_ as `0` value in flags. Such as `None`.
I would rather have something that is disabled.

Reply:
1) I removed `Unknown` in another PR (https://github.com/ravendb/ravendb/pull/9165/files#diff-950a76aa4eb57746fcc07663c915c377R5) and made `TrackChanges` option as `0` because it's our default and didn't want to allow users to choose `Unknown / None` which isn't a viable option anyway.

2) `Live / Passive` aren't really meaningful. Let me stay with current names as they are more descriptive I think.